Tools/scripts/build_options.py

```diff
@@ -262,6 +262,8 @@ def __init__(self,
     Feature('Actuators', 'FETTecOneWire', 'AP_FETTEC_ONEWIRE_ENABLED', 'Enable FETTec OneWire ESCs', 0, None),
 
     Feature('Payload', 'Camera', 'AP_CAMERA_ENABLED', 'Enable Camera Trigger support', 0, None),
+
+    Feature('Precision Landing', 'PrecLand', 'AP_PREC_LAND_ENABLED', 'Enable Precision Landing support', 0, None),
```

Comment:
```suggestion
    Feature('Precision Landing', 'PrecLand', 'AP_PRECLAND_ENABLED', 'Enable Precision Landing support', 0, None),
```

This makes it in-keeping with the way we usually form these defines based on the directory names.